src/components/TalentsListPage/SearchPanel/SearchPanel.jsx

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from "react";
+import styles from "./SearchPanel.module.scss";
+//import st from "../../Layout/Header/Header.module.scss";
+export function SearchPanel() {
+    const inputNames = useMemo(
```

Comment:
you dont need to store input names apart from inputs. they must be written in inputs